Slack.Automation/Promact.Core.Repository/ScrumRepository/ScrumBotRepository.cs

```diff
@@ -741,7 +747,7 @@ private bool IsPrivateChannel(string slackChannelId)
                             //all the scrum questions are answered as "leave"
                             foreach (Question question in questions)
                             {
-                                await AddUpdateAnswerAsync(scrumId, question.Id, expectedUserId, _stringConstant.Leave, ScrumAnswerStatus.Leave);
+                                await AddAnswerAsync(scrumId, question.Id, expectedUserId, _stringConstant.Leave, ScrumAnswerStatus.Leave);
```

Comment:
Here, you can create a collection and save it after foreach loop has completed. It will avoid lots of round trip